docs/pages/extensibility/CustomServerMiddleware.md

```diff
@@ -239,3 +239,18 @@ A custom middleware implementation needs to return a function with the following
 Custom middleware defining [Specification Version](../Configuration.md#specification-versions) 2.0 or higher have access to an interface of a [MiddlewareUtil](https://sap.github.io/ui5-tooling/v3/api/@ui5_server_middleware_MiddlewareUtil.html) instance.
 
 In this case, a `middlewareUtil` object is provided as a part of the custom middleware's [parameters](#custom-middleware-implementation). Depending on the specification version of the custom middleware, a set of helper functions is available to the implementation. The lowest required specification version for every function is listed in the [MiddlewareUtil API reference](https://sap.github.io/ui5-tooling/v3/api/@ui5_server_middleware_MiddlewareUtil.html).
+
+## Integration with `karma-ui5`
+
+!!! Warning
+    **Note:** The Karma project has been [deprecated](https://github.com/karma-runner/karma#karma-is-deprecated-and-is-not-accepting-new-features-or-general-bug-fixes) as of 2023
```

Comment:
```suggestion
!!! Warning
    The Karma project has been [deprecated](https://github.com/karma-runner/karma#karma-is-deprecated-and-is-not-accepting-new-features-or-general-bug-fixes) as of 2023
```

Reply:
I'm not terribly familiar with MD, but Warning and Note seem to be distinct entities. Do we still need the **Note** when we have a Warning?

Reply:
Thanks! I agree, we don't need both here.